docs/risk/risk-faq.md

```diff
@@ -16,13 +16,13 @@ Yes, EigenLayer does reuse the same economic security by enabling different AVSs
 
 Pooling allows various AVSs to contribute to a larger, shared economic security base, enhancing economic safety. This is because the cost to compromise any individual AVS increases significantly. For instance, consider 100 AVSs each with \$1B in economic security versus 100 AVSs sharing \$100B in economic security. The cost to attack any single AVS in the pooled system has effectively multiplied by 100.
 
-In addition to pooled security, EigenLayer lets each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and can only be slashed by one AVS. This arrangement guarantees AVS customers unconditional safety. 
+In addition to pooled security, EigenLayer will let each AVS acquire additional attributable security. Unlike pooled security, attributable security is specific to each AVS and will only be slashable by one AVS. This arrangement can provide game-theoretic guarantees for AVS customers. 
 
 Imagine a bridge protocol with a shared security base of \$10B and an attributable portion of \$10B. If a malfunction occurs, the protocol can assure the AVS customer that up to \$10B can still be securing the bridge through the attributable part. Attributable security also enables the flexible scaling of economic security. As an AVS, you can adjust your security budget based on user demand, catering to varying user needs at different times.
 
-Furthermore, EigenLayer achieves economies of scale by allowing AVSs to share the same underlying smart contract infrastructure. If a DeFi application uses services from five different AVSs and requests \$100B in unconditional safety, these AVSs can collectively purchase attributable security instead of each buying \$100B individually. This is feasible because if any of them fail, the \$100B could be compensated to the DeFi protocol.
+Furthermore, EigenLayer achieves economies of scale by allowing AVSs to share the same underlying smart contract infrastructure. If a DeFi application uses services from five different AVSs and requests \$100B in economic security, these AVSs can collectively purchase attributable security instead of each buying \$100B individually. This is feasible because if any of them fail, the \$100B could be compensated to the DeFi protocol.
```

Comment:
> achieves economies of scale

"aims to achieve"

Reply:
> these AVSs can collectively purchase attributable security instead of each buying \$100B individually

shouldn't this be "pooled security"?

Reply:
the wording of most of this paragraph is unclear

Reply:
> "aims to achieve"
https://github.com/Layr-Labs/eigenlayer-docs/pull/59/commits/3f8b91a2e642fa3f17b0d76006629683b729108d

> shouldn't this be "pooled security"?
I think it's valid with either pooled or attributable. pretty sure it was written with the mental model of them buying a total amount of attributable security and then some pooling of attributable security on top of that.